lib/buttons-and-links/__tests__/k-button.spec.js

```diff
@@ -0,0 +1,5 @@
+describe("KButton", () => {
```

Comment:
Is this part of this issue?

Reply:
:man_facepalming: no will remove and push again later on - thanks for catching that!